docs/changes.rst

```diff
@@ -29,6 +29,23 @@ Released: not yet
   raised in earlier versions may still be raised for other, less common cases.
   For details, see the corresponding item in the Enhancements section, below.
 
+* Changed 'SubscriptionManager.add_filter()' method to use the
+  'SourceNamespaces' property (allows multiple namespaces) of the
+  'CIM_IndicationFilter' class instead of the deprecated 'SourceNamespace'
+  property (allows only single namespace).  This changed the name of the
+  positional 'source_namespace' parameter to source_namespaces. The new
+  parameter allows both string and list of strings as values.
+
+  This change is brings the subscription manager in line with the the changes
+  made to this CIM class definition before DMTF CIM schema release 2.22.0.
```

Comment:
On "the changes made [...] before DMTF CIM schema release 2.22.0.": Talking about the changes does not go well with talking about "before" a particular release. Either it was the state (or something) before, or the "changes in".

Reply:
Modified grammar.  However, we do not know what version exactly because none of the older schema versions are onthe DMTF web page.  The 2.22.0 in OpenPegasus shows the new property and that is my only source of information 2.22.0 itself.